packages/scripts/src/launch-http-server.ts

```diff
@@ -0,0 +1,50 @@
+import express from 'express';
+import cors from 'cors';
+import { safeListeningHttpServer } from 'create-listening-server';
+import io from 'socket.io';
+import type { Socket } from 'net';
+
+export const DEFAULT_PORT = 3000;
+
+const noContentHandler: express.RequestHandler = (_req, res) => {
+    res.status(204); // No Content
+    res.end();
+};
+
+export async function launchHttpServer({
+   outputPath,
```

Comment:
maybe don't call it `outputPath` but `staticDirPath` since in this context it's just a path to a directory you statically serve